protobuf-test/src/common/v2/test_serde_derive.rs

```diff
@@ -0,0 +1,17 @@
+use super::test_serde_derive_pb::*;
+use serde_json;
+
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+struct Proto(TestSerde);
```

Comment:
Not clear how `struct Proto` is part of the test.

Reply:
I was originally "testing" by just verifying that we could implement Serialize and Deserialize (it would fail if TestSerde wasn't serializable.. but now I'm going with a more explicit test function.

Reply:
I was originally "testing" by just verifying that we could implement Serialize and Deserialize (it would fail if TestSerde wasn't serializable.. but now I'm going with a more explicit test function.